tests/conftest.py

```diff
@@ -243,6 +254,194 @@ def func_from_csv():
     return func
 
 
+@pytest.fixture
+def mc_env(example_env):
+    """Create an object to be used as the McEnvironment fixture in the tests.
+
+    Parameters
+    ----------
+    example_env : Environment
+        An object of the Environment class
+
+    Returns
+    -------
+    McEnvironment
+        An object of the McEnvironment class
+    """
+
+    # modify the environment to have a non-zero wind
+    example_env.windX = 3
+    example_env.windY = 4
+
+    return McEnvironment(
+        environment=example_env,
+        railLength=0.0005,
+        windXFactor=(1.0, 0.33, "normal"),
+        windYFactor=(1.0, 0.33, "normal"),
+    )
+
+
+@pytest.fixture
+def mc_solid_motor(solid_motor):
+    """Create an object to be used as the McSolidMotor fixture in the tests.
+
+    Parameters
+    ----------
+    solid_motor : SolidMotor
+        An object of the SolidMotor class
+
+    Returns
+    -------
+    McSolidMotor
+        An object of the McSolidMotor class
+    """
+    return McSolidMotor(
+        solidMotor=solid_motor,
+        burnOutTime=(3.9, 0.5),
+        grainsCenterOfMassPosition=0.001 / 10,
+        grainDensity=50,
+        grainSeparation=1 / 1000,
+        grainInitialHeight=1 / 1000,
+        grainInitialInnerRadius=0.1 / 1000,
+        grainOuterRadius=0,
+        totalImpulse=(3000, 30, "normal"),
+        nozzlePosition=0.001,
+    )
+
+
+@pytest.fixture
+def mc_rocket(rocket, mc_solid_motor):
+    """Create an object to be used as the McRocket fixture in the tests.
+
+    Parameters
+    ----------
+    example_rocket : Rocket
+        An object of the Rocket class
+
+    Returns
+    -------
+    McRocket
+        An object of the McRocket class
+    """
+
+    mc_rocket = McRocket(
+        rocket=rocket,
+        radius=0.001,
+        mass=0.001,
+        inertiaI=0.0300,
+        inertiaZ=0.0001,
+        powerOffDragFactor=(1, 0.033),
+        powerOnDragFactor=(1, 0.033),
+    )
+
+    nose_cone = rocket.addNose(
+        length=0.55829, kind="vonKarman", position=0.71971 + 0.558291
+    )
+
+    mc_nose_cone = McNoseCone(
+        nosecone=nose_cone,
+        length=0.001,
+    )
+
+    fin_set = rocket.addTrapezoidalFins(
+        4, span=0.100, rootChord=0.120, tipChord=0.040, position=-1.04956
+    )
+
+    mc_fin_set = McTrapezoidalFins(
+        trapezoidalFins=fin_set,
+        rootChord=0.0005,
+        tipChord=0.0005,
+        span=0.0005,
+    )
+
+    tail = rocket.addTail(
+        topRadius=0.0635, bottomRadius=0.0435, length=0.060, position=-1.194656
+    )
+
+    mc_tail = McTail(tail=tail, bottomRadius=0.0005, length=0.001)
+
+    def mainTrigger(p, y):
```

Comment:
I lost the count of how many times we define this function in the tests; Maybe we should convert it to a fixture as well!

Reply:
Hmmm... maybe this should be a change done in #345. With the changes in that PR we should be able to just remove these functions

Reply:
That's not the most important point actually. Currently a lot of tests repeat code when we need to add aerosurfaces to them, as well as the parachutes. Maybe we should create parachutes and surfaces as fixtures and avoid defining it inside different tests. Example:
- https://github.com/RocketPy-Team/RocketPy/blob/beta/v1.0.0/tests/test_flight.py#L108
- https://github.com/RocketPy-Team/RocketPy/blob/beta/v1.0.0/tests/test_flight.py#L199
- https://github.com/RocketPy-Team/RocketPy/blob/beta/v1.0.0/tests/test_flight.py#L400
- etc.

A new pull request in the future should modify the tests to avoid repeating code. I think I will open an issue with that regards soon.